model/src/main/scala/za/co/absa/atum/model/utils/JsonSyntaxExtensions.scala

```diff
@@ -36,16 +36,20 @@ object JsonSyntaxExtensions {
 
   implicit class JsonDeserializationSyntax(jsonStr: String) {
     def as[T: Decoder]: T = {
-      decode[T](jsonStr) match {
+      asSafe[T] match {
         case Right(value) => value
-        case Left(error) => throw new RuntimeException(s"Failed to decode JSON: $error")
+        case Left(error) => throw error
```

Comment:
why removing the prefix of the exception msg? I think it was quite useful, we would exactly know what this is coming from